components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt

```diff
@@ -592,14 +591,16 @@ internal class DisplayToolbar(
                 measuredHeight
             )
 
-            trackingProtectionAndSecurityIndicatorSeparatorView.layout(
-                trackingProtectionWidth,
-                0,
-                separatorWidth + trackingProtectionWidth,
-                measuredHeight
-            )
+            if(shouldSeparatorBeVisible) {
```

Comment:
nit: There's a space missing here after `if`. Did ktlint not complain? 🤔

Reply:
Sorry my bad. Fixed ✅